### PR TITLE
Update crosstargeting_override.props.sample

### PR DESCRIPTION
**Pull request by agneszitte-nventive**
Add additional comments for the crosstargeting_override.props.sample file

## PR Type
What kind of change does this PR introduce?

- Documentation content changes

## What is the current behavior?
Missing pieces of information for the Update crosstargeting_override.props.sample file

## What is the new behavior?
Added new pieces of information for the Update crosstargeting_override.props.sample file

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Contains **NO** breaking changes

### CODE REVIEW

File: src/crosstargeting_override.props.sample [hunk 1]
```diff
@@ -1,25 +1,40 @@
...
-      Note that if the BuildTargetFramework is not part of a project's 
-      TargetFrameworksDefault list, all target frameworks will be built.
-		-->
+    	This property controls the platform built by Visual Studio.
+		
+      	Available build targets:
+      				uap10.0 (Windows)
+				xamarinios10 (iOS)
+				MonoAndroid80 (Android 8.0)
+				MonoAndroid71 (Android 7.1)
```
Comment (jeromelaban) [inline src/crosstargeting_override.props.sample]: This one is not available for Uno.UI. It's 8.0 and higher.

File: src/crosstargeting_override.props.sample [hunk 2]
```diff
@@ -1,25 +1,40 @@
...
+     	TargetFrameworksDefault list, all target frameworks will be built.
+			
+    	To build multiple targets, separate them with ; (e.g. uap10.0;net46)
+    -->
     <BuildTargetFramework>MonoAndroid80</BuildTargetFramework>
 
     <!-- 
     This property allows the override of the nuget local cache.
     Set it to the version you want to override, used in another app.
+    The packages are located under this directory: "%USERPROFILE%\.nuget\packages"
```
Comment (jeromelaban) [inline src/crosstargeting_override.props.sample]: you can add `You will see the override path in the build output`

File: src/crosstargeting_override.props.sample (hunk 1 above)
Comment (agneszitte-nventive) [reply src/crosstargeting_override.props.sample]: Thanks will remove it !

File: src/crosstargeting_override.props.sample (hunk 2 above)
Comment (agneszitte-nventive) [reply src/crosstargeting_override.props.sample]: Thanks will add it

**Pull request merged**